API.md

```diff
@@ -0,0 +1,71 @@
+# OpenRailwayMap-API
+
+ OpenRailwayMap as an online service has different channels from which information can be gathered. One is the [OpenRailwayMap website](https://www.openrailwaymap.org/) to visit the map, another is the [OpenRailwayMap-API](https://api.openrailwaymap.) to retrieve INFORMATION about facilities, milestone positions or the network length for different railway infrastucture operators.
+
+## How is it programmed
+
+ The API uses JavaScript to extract the information brought in the URL. Depending on the request type (MODULE in the example) it loads the module js script. For further information visit the source code on [Github](https://github.com/OpenRailwayMap/OpenRailwayMap/).
+
+---
+
+## Use
+
+[https://api.openrailwaymap.org/MODULE?QUERY](https://api.openrailwaymap.org/networklength)
+
+ Examples: To receive information about the station Hamburg Hbf (main station) in Germany you can type:
+[https://api.openrailwaymap.org/facility?name=Hamburg Hbf](https://api.openrailwaymap.org/facility?name=Hamburg Hbf)
```

Comment:
The spaces should probably be turned into '+' or '%20'.

Reply:
Both work in the URL (in my setup), but it's changed